packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -363,6 +364,15 @@ export function TransferPanelMain({
     ? destinationAddressOrWalletAddress
     : walletAddress
 
+  const isTeleportMode = useMemo(
```

Comment:
The same thing is used a lot, maybe we should make a custom hook?

Reply:
Updated in all places